packages/components/loading-spinner/README.md

```diff
@@ -1,24 +1,46 @@
+<!-- THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. -->
+<!-- This file is created by the `yarn generate-readme` script. -->
+
 # LoadingSpinner
 
 ## Description
 
 A spinner animation to show the user some information is loading
 
+## Installation
+
+```
+yarn add @commercetools-uikit/loading-spinner
+```
+
+```
+npm --save install @commercetools-uikit/loading-spinner
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react
+```
+
+```
+npm --save install react
+```
+
 ## Usage
 
-```js
+```jsx
+import React from 'react';
 import LoadingSpinner from '@commercetools-uikit/loading-spinner';
 
-<LoadingSpinner size="s">{'Loading'}</LoadingSpinner>;
+const Example = () => <LoadingSpinner size="s">Loading</LoadingSpinner>;
+
+export default Example;
 ```
 
 ## Properties
 
-| Props      | Type     | Required | Values   | Default | Description                                         |
-| ---------- | -------- | :------: | -------- | ------- | --------------------------------------------------- |
-| `scale`    | `string` |          | 's', 'l' | `l`     | Defines the size of the loading spinner's container |
-| `children` | `string` |          | -        | -       | Used as the text for the loader                     |
-
-Main Functions and use cases are:
```

Comment:
Are we supporting this section somehow in the generation?

Reply:
This could be part of the description. good catch

Reply:
the **description** section is clear enough, so I'm thinking this information is a bit redundant.
I'll see what I can do, since the description is weird.